library/LuaApi.cpp

```diff
@@ -1617,6 +1617,45 @@ static const luaL_Reg dfhack_gui_funcs[] = {
     { NULL, NULL }
 };
 
+static void imgui_debug()
+{
+    ImGui::Begin("Hello");
+
+    ImGui::Text("Hello there!");
+
+    ImGui::End();
+}
+
+static bool imgui_begin(std::string title)
+{
+    return ImGui::Begin(title.c_str(), nullptr, 0);
+}
+
+static void imgui_end()
+{
+    ImGui::End();
+}
+
+static void imgui_text(std::string str)
+{
+    ImGui::Text("%s", str.c_str());
+}
+
+static bool imgui_button(std::string name)
```

Comment:
Note that simple intermediate functions like this are not necessary. You could just WRAPM the original module API

Reply:
Aha ok! I'll keep this in mind for simple stuff - things like buttons do have default arguments that I'll add down the line